crates/sequencer/src/sequencer.rs

```diff
@@ -464,12 +474,22 @@ where
             last_finalized_height
         );
 
-        let l1_fee_rate = self
+        let fee_rate_range = self.get_l1_fee_rate_range()?;
+
+        let mut l1_fee_rate = self
```

Comment:
rust allows "redefining" a variable. Remove this let and create a new l1_fee_rate below